tests/Unit/Controller/ThreadControllerTest.php

```diff
@@ -237,4 +237,48 @@ public function testDeleteTrash(): void {
 
 		$this->assertEquals(Http::STATUS_OK, $response->getStatus());
 	}
+
+	public function testSummarizeThread(): void {
+		$mailAccount = new MailAccount();
+		$mailAccount->setId(1);
+		$this->accountService
+			->method('find')
```

Comment:
nit: if you add `->with(...)` you can assert it's looking for the expected id

Reply:
you mean ->with() on the accountService?

Reply:
yes

Reply:
``$this->accountService->expect(self::once())->method('find')->with(1)->willReturn(new Account($mailAccount));``

then you know
1) it's been called exactly once
2) it's been called with the expected inputs